### ISSUE TITLE
Specifying 'height' using 'hv' behaviour changed

### ISSUE THREAD

**Issue by uvedhe** [jbkunst/highcharter#672]
Hi @jbkunst,

I found what I think is a bug: in a rather complex Shiny UI I want to specify chart height according to the page height, hence I've been working with `hv` (as in `height = "30vh"`). Since a recent update of `highcharter`, this now breaks the chart. 

Compare

```
highchart( height="30vh" ) %>% 
   hc_add_series(data.frame(x=c(1,2,3,4,5), 
                            y=c(6,3,23,6,4)), 
                 type="column")
```

to

```
highchart( ) %>% 
   hc_add_series(data.frame(x=c(1,2,3,4,5), 
                            y=c(6,3,23,6,4)), 
                 type="column")
```

If I increase the value to, say `300vh`, it shows the chart, but `vh` normally ranges from 0-100 only. Or is there a different logic for `highcharter` here?

Thanks!

EDIT: it turns out that `100vh` still works, but that it only affects the size of the outer container (which is now 100% of viewport height), while the graph *within* is squeezed to a tiny size (so perhaps a scaling bug?)

**Comment by uvedhe** [comment]
EDIT: it turns out that `100vh` still works, but that it only affects the size of the outer container (which is now 100% of viewport height), while the graph *within* is squeezed to a tiny size (so perhaps a scaling bug?)

**Comment by nordicgit70** [comment]
Hi, there was a change for issue #611 which was resolved in release 0.8.2.

In our Shiny app  using Highcharter the chart height is working fine (was not before). The chart size depends on the amount of data and set dynamically with `highchart(height = (200 + (nrow(pos) * 15)))` or later using `hc_size(hc, height = (400 + (100 * (length(hc$x$hc_opts$yAxis)))))`. The container for the output is `highcharter::highchartOutput("symbol", height = "auto")`.

Previously the issue was that the height properties were not used by the ShinyRender (see #611).  

Can it be related to how you set the height of the container?
